server/src/internalClusterTest/java/org/elasticsearch/snapshots/DedicatedClusterSnapshotRestoreIT.java

```diff
@@ -562,11 +561,8 @@ public void testRestoreIndexWithMissingShards() throws Exception {
 
     public void testRestoreIndexWithShardsMissingInLocalGateway() throws Exception {
         logger.info("--> start 2 nodes");
-        Settings nodeSettings = Settings.builder()
```

Comment:
These are unnecessary, we already set this rebalance to none in the parent (`AbstractSnapshotIntegTestCase`)